content/sensu-go/5.2/installation/verify.md

```diff
@@ -48,21 +48,21 @@ curl -LO https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.2.1/sensu-enterp
 Download Sensu for Windows.
 
 {{< highlight text >}}
-Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.2.1/sensu-go-5.2.1-windows-amd64.tar.gz  -OutFile C:\Users\Administrator\sensu-go-5.2.1-windows-amd64.tar.gz
+Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.2.1/sensu-go-5.2.1-windows-amd64.tar.gz  -OutFile "$env:userprofile\sensu-go-5.2.1-windows-amd64.tar.gz"
```

Comment:
With the package naming changes in 5.2.0, I believe the link should actually be:

```suggestion
Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.2.1/sensu-enterprise-go_5.2.1_windows_amd64.tar.gz  -OutFile "$env:userprofile\sensu-go-5.2.1-windows-amd64.tar.gz"
```